configure.sh

```diff
@@ -24,9 +24,11 @@ set -e # error out if required
 SCRIPT_EXEC="$(realpath $0)"
 MY_DIR="$(dirname $SCRIPT_EXEC)"
 
+echo "[$0] TARGET [${TARGET}]"
```

Comment:
TARGET in this case is $1 coming in

`TARGET=${1}`

Exit if $1 is not set.

Reply:
`echo "[$0] TARGET [${TARGET}]"` TARGET here is only working because of the export from the makefile... You should be in more control without the hidden transfer of TARGET.